tests/ut/test_core_setup.py

```diff
@@ -0,0 +1,268 @@
+from unittest import mock
+
+from cterasdk import toxmlstr
+from cterasdk import exception
+from cterasdk.core import setup
+from cterasdk.core.enum import ServerMode, SlaveAuthenticaionMethod, SetupWizardStage, SetupWizardStatus
+from cterasdk.common import Object
+from tests.ut import base_core
+
+
+class TestCoreSetup(base_core.BaseCoreTest):  # pylint: disable=too-many-instance-attributes
+
+    def setUp(self):
+        super().setUp()
+        self._master_ipaddr = '8.8.8.8'
+        self._master_secret = 'password'
+        self._admin_username = 'admin'
+        self._admin_email = 'admin@ctera.com'
+        self._admin_first_name = 'Service'
+        self._admin_last_name = 'Account'
+        self._admin_password = 'password'
+        self._domain = 'ctera.me'
+        self._replication_candidates = ['objs/6//Server/server', 'objs/7//Server/server1', 'objs/8//Server/server3']
+        self._replicate_from = 'server1'
+        self._array_of_statuses = []
+
+    def test_init_master(self):
+        self.patch_call("time.sleep")
+        self._init_global_admin()
+        self._array_of_statuses = [
+            TestCoreSetup._generate_status_response(SetupWizardStage.Server, SetupWizardStatus.NA, ''),
+            TestCoreSetup._generate_status_response(SetupWizardStage.Portal, SetupWizardStatus.NA, ''),
+            TestCoreSetup._generate_status_response(SetupWizardStage.Finish, SetupWizardStatus.NA, '')
+        ]
+        get_function = mock.MagicMock(side_effect=self._create_get_setup_status_response())
```

Comment:
You don't need another variable. You can just assign self._globa_admin.get = mock.MagicMock